static/usage/v7/modal/can-dismiss/child-state/angular/example_component_ts.md

```diff
@@ -18,7 +30,8 @@ export class ExampleComponent {
     this.presentingElement = document.querySelector('.ion-page');
   }
 
-  onDismissChange(canDismiss: boolean) {
+  onDismissChange(canDismiss: any) {
+    console.log('canDismiss', canDismiss);
```

Comment:
I think the answer is yes, but just to double-check, is the log intended?

Reply:
No this was unintentional, good catch! Fixed: https://github.com/ionic-team/ionic-docs/pull/3947/commits/4bd97d1b53e6a6a76cf0becd200762219ddcd061